### PR TITLE
MM-48120/MM-48623 Add ExperimentalSettings.PatchPluginsReactDOM to the System Console

### PR DESCRIPTION
**Pull request by hmhealey**
See the corresponding server PR for more information on what this setting actually does

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48120
https://mattermost.atlassian.net/browse/MM-48623

#### Related Pull Requests
https://github.com/mattermost/mattermost-server/pull/21171

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

File: e2e/playwright/support/server/default_config.ts [hunk 1]
```diff
@@ -529,6 +529,7 @@ const defaultServerConfig: AdminConfig = {
         EnableSharedChannels: false,
         EnableRemoteClusterService: false,
         EnableAppBar: false,
+        PatchPluginsReactDOM: false,
```
Comment (M-ZubairAhmed) [inline e2e/playwright/support/server/default_config.ts:532]: should we add the type in `packages/types/src/config.ts` too?

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by nevyangelova — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

File: e2e/playwright/support/server/default_config.ts (hunk 1 above)
Comment (hmhealey) [reply e2e/playwright/support/server/default_config.ts:532]: I added that in the previous commit. I just forgot to add it here in the latest one

**Pull request merged**